packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/plugin-ledger-connector-quorum.ts

```diff
@@ -111,7 +114,7 @@ export class PluginLedgerConnectorQuorum
     );
     this.web3 = new Web3(web3Provider);
     this.instanceId = options.instanceId;
-    this.pluginRegistry = options.pluginRegistry;
+    this.pluginRegistry = options.pluginRegistry as PluginRegistry;
```

Comment:
@TonyRowntree The plugin registry itself does not need to be optional because the API server is guaranteed to inject it to all plugin constructor options anyway (even if the options interface of a plugin doesn't actually ask for it, it's always there at runtime).